src/framework/audio/main/audiomodule.cpp

```
@@ -39,10 +39,12 @@
 
 #include "internal/audioconfiguration.h"
 #include "internal/audioactionscontroller.h"
+#include "internal/transporteventscontroller.h"
 #include "internal/audiouiactions.h"
 #include "internal/startaudiocontroller.h"
 #include "internal/playback.h"
 #include "internal/audiodrivercontroller.h"
```

Cppcheck (2.20.0)

[style] The function 'milisecsToSecs' is never used.

(unusedFunction)